package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@silexlabs/silex",
   "description": "Free and easy website builder for everyone.",
-  "version": "3.0.0-alpha.158",
+  "version": "3.0.0-alpha.159",
```

Comment:
Don't increment it by hand, it will be done when a new release is started

Reply:
I think it automatically incremented, I haven't touched it (maybe because I did ``npm run build`` to run tests). I'm changing this.

Reply:
please keep it to the original
at least don't commit it